site/docs/src/json/jwt/vend-post-response.json

```diff
@@ -0,0 +1,3 @@
+{
+  "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6ImRjMzBmYjRhOCJ9.eyJleHAiOjE2Mjk0ODY3MjQsImlhdCI6MTYyOTQ4MzEyNCwianRpIjoiZWU5YTAwZDktNTMyNS00YzYwLThjZjItMGQ5YzZlMWNhNDkwIiwiTmVvIjoiSSBrbm93IEt1bmcgRnUuIiwiTW9ycGhldXMiOiJTaG93IG1lLiJ9.8yfeCLfPL9MsrfjM50n4gT1wKK2ByEo7zINEVAdM9Hc"
```

Comment:
it would be nice to have this match up with the values in the `vend-post-request`. I liked the easter egg, but would prefer the output to correspond with the input (yea, I'm no fun).

Reply:
Well, at least one person saw it :). I agree if you decoded it then likely others will too and it will be better if it matches.